dev/drivers/scripts/aqm/prep/jevs_aqm_prep.sh

```diff
@@ -55,21 +53,7 @@ cd $DATA
 
 export cycle=t${cyc}z
 
-##
-## Instruction for Pull-Request testing
-## point COMIN to personal directory
-## output can be found at $COMOUT
-##
 export COMIN=/lfs/h2/emc/vpppg/noscrub/$USER/${NET}/${evs_ver}
-##
-## For aqmv7 NRT runs
-## export fcst_input_ver=v7
-## export COMINaqm=/lfs/h2/emc/ptmp/jianping.huang/emc.para/com/aqm/v7.0
-##
-## For AQMv6 restrospective output
-## export COMINaqm=/lfs/h2/emc/physics/noscrub/ho-chun.huang/verification/${MODELNAME}/${envir}
-##
-export COMOUT=$COMIN/${STEP}/${COMPONENT}
 
```

Comment:
Can you please bring COMOUT back into this driver? I suggest:
`export COMOUT=/lfs/h2/emc/vpppg/noscrub/${USER}/${NET}/${evs_ver}/${STEP}/${COMPONENT}/${RUN}`

Reply:
Done.